tests/CMakeLists.txt

```
@@ -44,9 +44,10 @@ set_tests_properties(touches_conversion_v2 parquet_conversion_v2
 
 add_executable(test_indexing test_indexing.cpp
                              ${${PROJECT_NAME}_SOURCE_DIR}/src/index/index.cpp)
-target_link_libraries(test_indexing Catch2::Catch2WithMain HighFive MPI::MPI_C)
+target_link_libraries(test_indexing Catch2::Catch2WithMain HighFive MPI::MPI_CXX)
```

```suggestion
target_link_libraries(test_indexing Catch2::Catch2WithMain HighFive MPI::MPI_C)
```
We don't use C++ MPI features normally, so linking against the C target should be sufficient. Did you encounter any problems using the C target?